components/SkilledImpact/elements/CampaignContent.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import styles from '../../../constants/SkilledImpact/OrgSkilledImpactBody';
+import Lightening from '../../../assets/jsicons/bottomnavigation/Lightening';
+import ChevronBottom from '../../../assets/jsicons/miscIcons/ChevronBottom';
+import ChevronRight from '../../../assets/jsicons/miscIcons/ChevronRight';
+import { Avatar } from 'react-native-elements';
+import * as moment from 'moment';
+
+class CampaignContent extends React.Component {
+  constructor(props) {
+    //TODO props edits and states
+    super(props);
+    this.state = {
+      skillExpand: true,
+      isReachable: true,
```

Comment:
**question**: Are we using this here? Or is this just accidentally copy-pasted from the other box?